src/internal/tracing/extended/extended_trace.proto

```diff
@@ -11,6 +11,7 @@ message TraceProto {
   // (trace ID, span ID, sampled) tuple, basically)
   map<string, string> serialized_trace = 1;
 
+  string project = 3;
```

Comment:
i guess it probably would have been nicer if we just had the Pipeline proto in here

Reply:
Agreed, but since this already had the string name instead of the message I figured that it was better to make the small change than a larger one. Could have gone either way, TBH.